examples/adb-exfiltration-protection/variables.tf

```diff
@@ -15,6 +15,22 @@ variable "spokecidr" {
   default     = "10.179.0.0/20"
 }
 
+variable "existing_resource_group_name" {
+  type        = string
+  description = "Specify the name of an existing Resource Group only if you do not want Terraform to create a new one"
+  validation {
+    condition     = var.create_resource_group == true || length(var.existing_resource_group_name) > 0
```

Comment:
If we do that, we need to bump minimum Terraform version as it was introduced in 1.9: https://www.hashicorp.com/en/blog/terraform-1-9-enhances-input-variable-validations